src/redux/utilities.js

```diff
@@ -10,8 +10,10 @@ export default function formatContentType(contentType) {
     onlinereference: 'Online Reference'
   };
 
-  if (contentType) {
-    return contentTypes[contentType.toLowerCase()];
+  let contentTypeKey = contentType.toLowerCase();
+
+  if (contentType && Object.prototype.hasOwnProperty.call(contentTypes, contentTypeKey)) {
```

Comment:
Curious, why not just `contentTypes.hasOwnProperty(contentTypeKey)`?

Reply:
We have this turned on https://eslint.org/docs/rules/no-prototype-builtins

Reply:
Gotcha